test/scenario/metering-plans/standard-services-hours/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "abacus-services-hours-scenario-test",
+  "description": "Services hours scenario tests",
+  "license": "Apache-2.0",
+  "version": "1.1.3",
+  "private": true,
+  "homepage": "https://github.com/cloudfoundry-incubator/cf-abacus/test/acceptance/standard-services-hours-test/",
```

Comment:
You are missing a `tree/master` section in the URL. If you try to open your URL address, it will result in `404`.